### ISSUE TITLE
Vue.js style lang attribute

### ISSUE THREAD

**Issue by petrprikryl** [biomejs/biome#7917]
Hi, using `<style lang="scss">` gives me:

```
  ✖ Invalid document: Expected end tag of kind Embedded but found Group.
                                                                                                                                                                                                                                                                                                                                                                                                                         
     This is an internal Biome error. Please report if necessary.                                                                                                                                                                                                                                                                                                                                                        
```  

I know that sass is not supported but could be style block just skipped in this case?

For reference: https://vite.dev/guide/features.html#css-pre-processors:~:text=also%20automatically%20enables-,%3Cstyle%20lang%3D%22sass%22%3E,-et%20al.

### Environment information

```bash
CLI:
  Version:                      2.3.2
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           linux

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  BIOME_THREADS:                unset
  NO_COLOR:                     unset
  TERM:                         xterm
  JS_RUNTIME_VERSION:           v25.0.0
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         unset

Biome Configuration:
  Error:                        The configuration schema version does not match the CLI version 2.3.2
  Status:                       Loaded with errors
  Path:                         biome.json
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               true
  VCS enabled:                  true

Formatter:
  Format with errors:           unset
  Indent style:                 Space
  Indent width:                 unset
  Line ending:                  unset
  Line width:                   unset
  Attribute position:           unset
  Bracket spacing:              unset
  Includes:                     unset

JavaScript Formatter:
  Enabled:                      unset
  JSX quote style:              unset
  Quote properties:             unset
  Trailing commas:              unset
  Semicolons:                   unset
  Arrow parentheses:            unset
  Bracket spacing:              unset
  Bracket same line:            unset
  Quote style:                  Double Quotes
  Indent style:                 unset
  Indent width:                 unset
  Line ending:                  unset
  Line width:                   unset
  Attribute position:           unset

JSON Formatter:
  Enabled:                      unset
  Indent style:                 unset
  Indent width:                 unset
  Line ending:                  unset
  Line width:                   unset
  Trailing Commas:              unset
  Expand lists:                 unset

CSS Formatter:
  Enabled:                      true
  Indent style:                 unset
  Indent width:                 unset
  Line ending:                  unset
  Line width:                   unset
  Quote style:                  Double Quotes

GraphQL Formatter:
  Enabled:                      unset
  Indent style:                 unset
  Indent width:                 unset
  Line ending:                  unset
  Line width:                   unset
  Bracket spacing:              unset
  Quote style:                  unset

Workspace:
  Open Documents:               0
```

### Configuration

```JSON
{
  "$schema": "https://biomejs.dev/schemas/2.3.0/schema.json",
  "vcs": {
    "enabled": true,
    "clientKind": "git",
    "useIgnoreFile": true
  },
  "files": {
    "ignoreUnknown": false,
    "includes": ["assets/**"]
  },
  "formatter": {
    "enabled": true,
    "indentStyle": "space"
  },
  "linter": {
    "enabled": true,
    "rules": {
      "recommended": true
    },
    "domains": {
      "vue": "recommended"
    }
  },
  "javascript": {
    "formatter": {
      "quoteStyle": "double"
    }
  },
  "html": {
    "experimentalFullSupportEnabled": true,
    "formatter": {
      "enabled": true,
      "indentWidth": 2
    }
  },
  "css": {
    "formatter": {
      "enabled": true,
      "quoteStyle": "double"
    },
    "parser": {
      "cssModules": true
    }
  },
  "assist": {
    "enabled": true,
    "actions": {
      "source": {
        "organizeImports": "on"
      }
    }
  }
}
```

### Playground link

https://biomejs.dev/playground/?code=PABzAGMAcgBpAHAAdAAgAHMAZQB0AHUAcAA%2BAAoAPAAvAHMAYwByAGkAcAB0AD4ACgAKADwAdABlAG0AcABsAGEAdABlAD4ACgAgACAAPABkAGkAdgA%2BAGYAbwBvADwALwBkAGkAdgA%2BAAoAPAAvAHQAZQBtAHAAbABhAHQAZQA%2BAAoACgA8AHMAdAB5AGwAZQAgAGwAYQBuAGcAPQAiAHMAYwBzAHMAIgA%2BAAoAYgB1AHQAdABvAG4AIAB7AAoAIAAgAHcAaQBkAHQAaAA6ACAAMgA1AHAAeAA7AAoAfQAKADwALwBzAHQAeQBsAGUAPgAKAA%3D%3D&language=vue

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by dyc3** [comment]
Right, we don't support scss at the moment. But this should at least have a better error/warning.

**Comment by ceferrari** [comment]
Same problem. I don't mind waiting for full SCSS support or a better warning, but I'd like to disable it only for `<style lang="scss">`, while keeping it enabled for the rest of the .vue file (`script` and `template` tags).

I have tried using `biome-ignore-start` and `biome-ignore-end` but it didn't work.

**Comment by ematipico** [comment]
That's where we parse the CSS 

https://github.com/biomejs/biome/blob/main/crates%2Fbiome_service%2Fsrc%2Ffile_handlers%2Fhtml.rs#L553-L554

We should add a function like `is_scss_lang` to the `HtmlElement`, so we can exclude the parsing.

It would work pretty much like `is_typescript_lang`

**Comment by lisiur** [comment]
<del>I don't think this is an error reported by SCSS. If the script doesn't have line breaks, Biome won't report an error.</del>

It seems that the line breaks in the script are somehow related to the SCSS judgment.

https://biomejs.dev/playground/?code=PABzAGMAcgBpAHAAdAAgAHMAZQB0AHUAcAA%2BADwALwBzAGMAcgBpAHAAdAA%2BAAoACgA8AHQAZQBtAHAAbABhAHQAZQA%2BAAoAIAAgADwAZABpAHYAPgBmAG8AbwA8AC8AZABpAHYAPgAKADwALwB0AGUAbQBwAGwAYQB0AGUAPgAKAAoAPABzAHQAeQBsAGUAIABsAGEAbgBnAD0AIgBzAGMAcwBzACIAPgAKAGIAdQB0AHQAbwBuACAAewAKACAAIAB3AGkAZAB0AGgAOgAgADIANQBwAHgAOwAKAH0ACgA8AC8AcwB0AHkAbABlAD4ACgA%3D&language=vue

**Comment by lisiur** [comment]
I just tested it, and if this part doesn't return None, there won't be an error. 👀

https://github.com/biomejs/biome/blob/b51968b1aeb16284a25a54a24b677f047f104506/crates/biome_service/src/file_handlers/html.rs#L560-L562

**Comment by ceferrari** [comment]
Any update on this?

**Comment by leondewit** [comment]
This is currently preventing us from using the HTML parser. Are there any plans to fix this issue?
